<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormValidasi</class>
 <widget class="QWidget" name="FormValidasi">
  <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>700</width> <height>600</height> </rect> </property>
  <property name="windowTitle"> <string>Form Validasi</string> </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry"> <rect> <x>50</x> <y>20</y> <width>450</width> <height>200</height> </rect> </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0"><widget class="QLabel" name="l1"><property name="text"><string>No Validasi</string></property></widget></item>
    <item row="0" column="1"><widget class="QLineEdit" name="editNoVal"/></item>
    <item row="1" column="0"><widget class="QLabel" name="l2"><property name="text"><string>ID Admin</string></property></widget></item>
    <item row="1" column="1"><widget class="QLineEdit" name="editIDAdmin"/></item>
    <item row="2" column="0"><widget class="QLabel" name="l3"><property name="text"><string>No Pendaftaran</string></property></widget></item>
    <item row="2" column="1"><widget class="QLineEdit" name="editNoDaftar"/></item>
    <item row="3" column="0"><widget class="QLabel" name="l4"><property name="text"><string>Tgl Validasi</string></property></widget></item>
    <item row="3" column="1"><widget class="QLineEdit" name="editTglVal"/></item>
    <item row="4" column="0"><widget class="QLabel" name="l5"><property name="text"><string>Catatan Tambahan</string></property></widget></item>
    <item row="4" column="1"><widget class="QLineEdit" name="editCatatan"/></item>
    <item row="5" column="0"><widget class="QLabel" name="l6"><property name="text"><string>Hasil</string></property></widget></item>
    <item row="5" column="1"><widget class="QLineEdit" name="editHasil"/></item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan"> <property name="geometry"> <rect> <x>60</x> <y>240</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Simpan</string> </property> </widget>
  <widget class="QPushButton" name="btnHapus"> <property name="geometry"> <rect> <x>170</x> <y>240</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Hapus</string> </property> </widget>
  <widget class="QPushButton" name="btnUbah"> <property name="geometry"> <rect> <x>280</x> <y>240</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Ubah</string> </property> </widget>
  <widget class="QPushButton" name="btnBersih"> <property name="geometry"> <rect> <x>390</x> <y>240</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Bersih</string> </property> </widget>
  <widget class="QTableWidget" name="tabelValidasi">
   <property name="geometry"> <rect> <x>50</x> <y>290</y> <width>600</width> <height>250</height> </rect> </property>
   <column><property name="text"><string>No Validasi</string></property></column>
   <column><property name="text"><string>No Daftar</string></property></column>
   <column><property name="text"><string>Tgl Val</string></property></column>
   <column><property name="text"><string>Hasil</string></property></column>
  </widget>
 </widget>
 <resources/> <connections/>
</ui>
